dockstore-integration-testing/src/test/java/io/dockstore/common/CommonTestUtilities.java

```diff
@@ -49,6 +49,7 @@
     public static final int WAIT_TIME = 60000;
 
 
+    public static final String PUBLIC_CONFIG_PATH_WITH_API = ResourceHelpers.resourceFilePath("dockstoreAPI.yml");
```

Comment:
Is the only difference the value of externalConfig basePath?
If so, I would just change the value of dockstore.yml across the board

Reply:
basePath and create => validate.  changing dockstore.yml to validate causes a whole lot of problems in other tests.  using the confidential yml when the ga4gh tests doesn't need credentials seems overkill.

Reply:
I don't think we should be using create in anything, so that may be worth removing/updating anyway. Let's go with that, I agree that the confidential one would be overkill.